integration/commands_administration_test.go

```diff
@@ -185,18 +185,69 @@ func TestCommandsAdministrationGetParameter(t *testing.T) {
 		databaseName: "admin",
 	})
 
-	var actual bson.D
-	err := collection.Database().RunCommand(ctx, bson.D{{"getParameter", "*"}}).Decode(&actual)
-	require.NoError(t, err)
+	for name, tc := range map[string]struct {
+		command    bson.D
+		expected   map[string]any
+		unexpected []string
+		err        *mongo.CommandError
+	}{
+		"AllParameters": {
+			command: bson.D{{"getParameter", "*"}},
+			expected: map[string]any{
+				"acceptApiVersion2": false,
+				"authSchemaVersion": int32(5),
+				"quiet":             false,
+				"ok":                float64(1),
+			},
+		},
+		"ExistingParameters": {
+			command: bson.D{{"getParameter", 1}, {"quiet", 1}},
+			expected: map[string]any{
+				"quiet": false,
+				"ok":    float64(1),
+			},
+		},
+		"NonexistentParameters": {
+			command: bson.D{{"getParameter", 1}, {"quiet", 1}, {"quiet_other", 1}},
+			expected: map[string]any{
+				"quiet": false,
+				"ok":    float64(1),
+			},
+			unexpected: []string{"quiet_other"},
+		},
+		"EmptyParameters": {
+			command: bson.D{{"getParameter", 1}},
+			err:     &mongo.CommandError{Code: 0, Name: "", Message: `no option found to get`},
+		},
+		"OnlyNonexistentParameters": {
+			command: bson.D{{"getParameter", 1}, {"quiet_other", 1}},
+			err:     &mongo.CommandError{Code: 0, Name: "", Message: `no option found to get`},
```

Comment:
```suggestion
			err:     &mongo.CommandError{Message: `no option found to get`},
```

Zero values are good enough

Reply:
fix

Reply:
hmmm, looks like there are some more similar lines of code, could you please fix them also?

Reply:
all tests are different.
can't find the same

Reply:
I mean, you wrote that it's fixed and it looks like you didn't push.  It is actually still in the code.